frontend/src/components/Overlay/LevelsComplete.tsx

```diff
@@ -10,35 +9,40 @@ function LevelsComplete({
 	goToSandbox: () => void;
 	closeOverlay: () => void;
 }) {
-	const managerDialogue = [
-		{
-			speaker: 'ScottBrew Manager',
-			text: `Congratulations on becoming our new Head of Security! 
-      You can now fully explore and adjust ScottBrewBot's system, defences, prompts and more.
-      Glad to finally have you in the right role.`,
-		},
+	const speakerArray = ['ScottBrew Manager', 'SpyLogic'];
+	const pages = [
+		<>
+			<h2>ScottBrew Manager:</h2>
+			<p>
+				Congratulations on becoming our new Head of Security! You can now fully
+				explore and adjust ScottBrewBot&apos;s system, defences, prompts and
+				more. Glad to finally have you in the right role
+			</p>
+		</>,
+		<>
+			<h2>You&apos;ve completed the story mode</h2>
+			<p>
+				You can stay here and continue to play with the levels, or you can move
+				onto Sandbox mode where you can configure your own defence set up and
+				try to break it.
+			</p>
+			<p>
+				You can always switch modes by clicking on the button in the left panel.
```

Comment:
It was about switching back and forth between sandbox and levels if I recall correctly. I didn't make any changes to the text, it's leftover from earlier versions of the game. 

Maybe that paragraph ("You can always switch modes by clicking on the button in the left panel.") can be entirely removed, since by the time you get the win modal, users will be familiar with the UI, the level buttons and the Getting Started button?